src/Reweighting/reweight.jl

```diff
@@ -0,0 +1,177 @@
+struct ReweightResults
+    probability::Vector{Float64}
+    relative_probability::Vector{Float64}
+    energy::Vector{Float64}
+end
+
+"""
+    reweight(simulation, f_perturbation, group_1, cutoff; prob = true)
```

Comment:
note que você colocou o cutoff como parâmetro aqui. Mas fica melhor se for um keyword parameter (depois do `;`).